### PR TITLE
Auto focus only on mount

### PR DESCRIPTION
**Pull request by bernhardoj**
### Details

Removed `autoFocus` from the `useEffect` deps so it only auto focus on mount.

### Related Issues

https://github.com/Expensify/App/issues/42645

### Manual Tests

1. Add an `autoFocus` state to the example app
2. Add a new button to toggle the state
3. Press the button to toggle the state
4. If the state is true, verify the input isn't focused

Before:

https://github.com/Expensify/react-native-live-markdown/assets/50919443/74f976c0-5c3b-46d8-aeb3-80099ccba400

After:

https://github.com/Expensify/react-native-live-markdown/assets/50919443/18330d30-1e78-4a57-9b3a-909f52895b0b

### CODE REVIEW

**Comment by tomekzaw** [comment]
@bernhardoj Thanks for the PR!

LGTM but let's wait for @Skalakid's or @BartoszGrajdek's approval prior to merging.

**Review by BartoszGrajdek — APPROVED** [review]
Makes sense, approved! ✅

**Review by Skalakid — APPROVED** [review]
LGTM

**Pull request merged**
